Tweeter/Scripts/app.js

```diff
@@ -2,6 +2,7 @@
 
     $("#username-ans").removeClass("glyphicon-ok");
     $("#username-ans").removeClass("glyphicon-remove");
+    $("input[type='submit']").removeClass('disabled');
     $.ajax({
```

Comment:
The only issue I can see here is that if the ajax call takes some time (which it normally wouldn't!) you would have the button flickering between enabled and disabled.